plugins/aws/resoto_plugin_aws/resource/cloudwatch.py

```diff
@@ -450,3 +450,27 @@ def query_for(
 
 
 resources: List[Type[AwsResource]] = [AwsCloudwatchAlarm, AwsCloudwatchLogGroup, AwsCloudwatchMetricFilter]
+
+V = TypeVar("V", bound=BaseResource)
+
+
+def update_resource_metrics(
+    resources_map: Dict[str, V],
+    cloudwatch_result: Dict[AwsCloudwatchQuery, AwsCloudwatchMetricData],
+    metric_normalizers: Dict[str, MetricNormalization],
+) -> None:
+    for query, metric in cloudwatch_result.items():
+        resource = resources_map.get(query.ref_id)
+        if resource is None:
+            continue
+        metric_value = next(iter(metric.metric_values), None)
+        if metric_value is None:
+            continue
+        normalizer = metric_normalizers.get(query.metric_name)
+        if not normalizer:
+            continue
```

Comment:
metric without normalizer should be thrown away? Is this expected?

Reply:
that's a neat way to unwrap options here, so yes.